test/source/test.ts

```diff
@@ -157,12 +157,7 @@ test.after.always('evaluate Catch.reportErr errors', async t => {
         !e.message.match(/Bad Request: 400 when GET-ing https:\/\/.*localhost:\d+\/flowcrypt-email-key-manager/)
     )
     // below for test "decrypt - failure retrieving chunk download - next request will try anew"
-    .filter(
-      e =>
-        !/400 when GET-ing https:\/\/.*localhost:\d+\/gmail\/v1\/users\/me\/messages\/1885ded59a2b5a8d\/attachments\/ANGjdJ_0g7PGqJSjI8-Wjd5o8HcVnAHxIk-H210TAxxwf/.test(
-          e.message
-        )
-    )
+    .filter(e => !/https:\/\/gmail\.localhost:[\d]*\/gmail\/1885ded59a2b5a8d/.test(e.url))
```

Comment:
This test `decrypt - failure retrieving chunk download - next request will try anew`

Reply:
@ioanmo226 looks like it fails because in updated code [describeApiAction](https://github.com/FlowCrypt/flowcrypt-browser/blob/c0fbb59b1c4cc61309d83465a74676f4593775b0/extension/js/common/api/shared/api-error.ts#L35) isn't used when formatting error message.

After fixing this one, I think we can merge PR into `master` and fix any other found issues later.

Reply:
Okay